Godeps/Godeps.json

```diff
@@ -8,10 +8,12 @@
 	"Deps": [
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
+			"Comment": "null-5",
```

Comment:
As long as our `hack/verify-*` scripts do not fall over them. We should have the ignore flag in the diff commands everywehere.